redis/redisClient_test.go

```diff
@@ -0,0 +1,3 @@
+package redis_test
+
+// TODO: add unit tests
```

Comment:
I do not have a clue how to unit test redisClient. Or rabbitClient. But since the structs are just glue code, we might system-test them once and we should not mess with them in the future. This is the text-book definition of the 3-rd party lib glue code. And that is the reason it should always be kept as thin as possible.

Reply:
:+1:

Reply:
my initial thought was to use some external libraries for that (like miniredis), but as you said, if we keep them as thin as possible, it's fine